java/src/main/java/ai/rapids/cudf/HostMemoryBuffer.java

```diff
@@ -109,6 +139,38 @@ public static HostMemoryBuffer allocate(long bytes) {
     return allocate(bytes, defaultPreferPinned);
   }
 
+  /**
+   * Create a host buffer that is memory-mapped to a file.
+   * @param path path to the file to map into host memory
+   * @param mode mapping type
+   * @param offset file offset where the map will start
+   * @param length the number of bytes to map
+   * @return file-mapped buffer
+   */
+  public static HostMemoryBuffer mapFile(String path, MapMode mode,
```

Comment:
nit: I personally would prefer to see path be a `File` or a `Path` just so it is 100% clear what this is mapping in, and it is not a URL or some kind.